e2e/playwright/file-tree.spec.ts

```diff
@@ -11,37 +12,115 @@ import {
 import { join } from 'path'
 import { FILE_EXT } from 'lib/constants'
 
-test.beforeEach(async ({ context, page }, testInfo) => {
+_test.beforeEach(async ({ context, page }, testInfo) => {
   await setup(context, page, testInfo)
 })
 
-test.afterEach(async ({ page }, testInfo) => {
+_test.afterEach(async ({ page }, testInfo) => {
   await tearDown(page, testInfo)
 })
 
+test.describe('integrations tests', () => {
+  test('Creating a new file or switching file while in sketchMode should exit sketchMode', async ({
+    tronApp,
+    homePage,
+    scene,
+    editor,
+    toolbar,
+  }) => {
+    await tronApp.initialise({
+      fixtures: { homePage, scene, editor, toolbar },
```

Comment:
This is the akward thing about how I went about getting the fixtures to work with electron, they have to be passed to the initialise function so they can have their `page` set again since it's different from the integrated playwright page.